src/main/java/frc/robot/vision/cameras/limelight/LimelightPipeline.java

```diff
@@ -1,15 +1,23 @@
 package frc.robot.vision.cameras.limelight;
 
+import frc.robot.vision.DetectedObjectType;
+import frc.utils.alerts.Alert;
+
+import java.util.Optional;
+
+
 public enum LimelightPipeline {
 
-	APRIL_TAG(0, false, true),
-	OBJECT_DETECTION(1, true, false);
+	APRIL_TAG(0, false, true, new DetectedObjectType[0]),
+	OBJECT_DETECTION(1, true, false, new DetectedObjectType[0]);
 
 	private final int pipelineIndex;
 	private final boolean isDetectingObjects;
 	private final boolean isUsingMT;
```

Comment:
```suggestion
	private final boolean detectsObjects;
	private final boolean usesMT;
```
the pipeline uses mt, its not using it right now

Reply:
doesUseMT is gramaticaly incorrect, and correcting it makes usesMT